src/basic/alloc-util.c

```diff
@@ -102,3 +102,7 @@ void* greedy_realloc0(
 
         return q;
 }
+
+void *expand_to_usable(void *ptr, size_t newsize _unused_) {
```

Comment:
this really should carry a comment explaining why this can't be a static inline function

Reply:
and maybe add a brief comment here, that this only exists as conduit for the `_alloc_` decoration, and serves no other purpose

Reply:
May I put the comment in the header instead, since that's where the `_alloc_` attribute is?